### PR TITLE
feat (User Management): add query params search and include_settings to list_users operation

### PR DESCRIPTION
**Pull request by PureMathematics**
## PR summary

Update Include_settings and substring search functionality (User Management) 

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information
the current issue for updating SDK https://github.ibm.com/IAM/uum/issues/5882 
the previous issue for the user management code change https://github.ibm.com/IAM/uum/issues/4948

### CODE REVIEW

File: examples/test_user_management_v1_examples.py [hunk 1]
```diff
@@ -186,6 +186,58 @@ def test_list_users_example(self):
         except ApiException as e:
             pytest.fail(str(e))
 
+    @needscredentials
+    def test_list_users_example_with_include_settings(self):
```
Comment (padamstx) [inline examples/test_user_management_v1_examples.py:190]: one code example per operation :)

**Review by padamstx — CHANGES_REQUESTED** [review]
Changes look good except for the same issues as the other PRs:
1. use git commit -s option
2. one code fragment per operation in the examples

Also, please run "make lint-fix" in your sandbox to reformat the source files and then commit those results (see update_service.md).

**Comment by PureMathematics** [comment]
integration test
python-integration

**Comment by PureMathematics** [comment]
examples test
python-example

**Pull request merged**